resources/qml/PrintMonitor.qml

```diff
@@ -20,49 +20,462 @@ Column
         simpleNames: true
     }
 
-    Item
+    Rectangle
     {
-        width: base.width - 2 * UM.Theme.getSize("default_margin").width
-        height: childrenRect.height + UM.Theme.getSize("default_margin").height
-        anchors.left: parent.left
-        anchors.leftMargin: UM.Theme.getSize("default_margin").width
+        id: connectedPrinterHeader
+        width: parent.width
+        height: childrenRect.height + UM.Theme.getSize("default_margin").height * 2
+        color: UM.Theme.getColor("setting_category")
 
         Label
         {
+            id: connectedPrinterNameLabel
+            text: printerConnected ? connectedPrinter.name : catalog.i18nc("@info:status", "No printer connected")
+            font: UM.Theme.getFont("large")
+            color: UM.Theme.getColor("text")
+            anchors.left: parent.left
```

Comment:
Done in 60812139b78c927b52b91c8264adaa8ff4bb7989.

Note that this will cause the width of the default margin to be used for vertical margin. However it seems we already do this in several places, so I'll follow suit.